.github/worflows/cmake.yml

```diff
@@ -0,0 +1,35 @@
+name: CMake-Manually triggered workflow
+
+on:
+  workflow_dispatch:
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+    runs-on: windows  
```

Comment:
is this purposefully only windows? or could this work for mac/linux? is so, you might want to consider making that an input to the workflow_dispatch

Reply:
a few adjustments will make them run in other environments. I want to test the windows config